videostore/videostore.go

```diff
@@ -61,8 +61,9 @@ type videostore struct {
 	config Config
 	logger logging.Logger
 
-	latestFrame atomic.Pointer[C.AVFrame]
-	workers     *utils.StoppableWorkers
+	latestFrame   *C.AVFrame
+	latestFrameMu sync.Mutex
```

Comment:
Using a normal lock instead of `sync.RWMutex` since ratio of reads to writes is near 1.